wordpress-mobile/WordPress-Android
# [Jetpack Social] Post settings UI fixes

### Refresh SiteModel when moving back from WebView
To test:
1 - Make sure `JetpackSocialFeatureConfig` feature flag is enabled;
2 - Select a SH site with Jetpack plugin or Jetpack Social plugin installed;
3 - Make sure you have at least one social sharing account connected;
4 - Open post list;
5 - Select a draft or create a new one;
6 - Tap the overflow menu and select "Post settings";
7 - Scroll down to the "Social Sharing" section: the share limit should be shown;
8 - Tap the `SUBSCRIBE TO SHARE MORE` button;
9 - Purchase the plan;
10 - Move back to post settings: the share limit should be gone;

### Fix share limit text color in dark theme
To test:
1 - Make sure `JetpackSocialFeatureConfig` feature flag is enabled;
2 - Select a SH site with Jetpack plugin or Jetpack Social plugin installed;
3 - Make sure you have at least one social sharing account connected;
4 - Open post list;
5 - Select a draft or create a new one;
6 - Tap the overflow menu and select "Post settings";
7 - Scroll down to the "Social Sharing" section;
8 - Observe the share limit description text: it should be visible in light theme and dark theme.

<img width="362" alt="image" src="https://github.com/wordpress-mobile/WordPress-Android/assets/14964993/584a8697-d9af-4e66-b2ed-99b5cd33c825">
<img width="362" alt="image" src="https://github.com/wordpress-mobile/WordPress-Android/assets/14964993/9e8536d0-fb75-4421-97d7-93e901712fa1">

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)


PR submission checklist:


UI Changes testing checklist:
